File: Learnify.Domain/Entities/Question.cs
using Learnify.Domain.Enums;
using System.Text.Json.Serialization;

namespace Learnify.Domain.Entities
{
    public class Question : BaseEntity
    {
        public string Text { get; set; } = string.Empty;
        public QuestionType Type { get; set; }
        public Guid QuizId { get; set; }
        [JsonIgnore] // Add this attribute
        public Quiz? Quiz { get; set; }
        public virtual ICollection<Answer>? Answers { get; set; }
    }
}

File: Learnify.UI/src/models/entities/Question.js
export class Question {
    constructor(
        id,
        text,
        type = null,
        quizId,
        quiz = null,
        answers = [],
    ) {
        this.id = id;
        this.text = text;
        this.type = type;
        this.quizId = quizId;
        this.quiz = quiz;
        this.answers = answers;
        
    }
}